src/create-entity-api.ts

```diff
@@ -90,6 +120,14 @@ export function createEntityApi<
           transformResponse: (response: object) => createEntityInstance<TEntity>(entityConstructor, response)
         }),
 
+        /**
+         * Creates a query endpoint that requests `GET /{baseEndpoint}` for searching entities.
+         * Accepts request params described by `PaginationRequest` and returns `PaginationResponse`.
+         * Both classes can be extended to define your onw structure using `entitySearchRequestConstructor` and `entitySearchResponseConstructor` in `createEntityApi`.
```

Comment:
```suggestion
         * Accepts request params described by `entitySearchRequestConstructor` and returns `entitySearchResponseConstructor` extending `PaginationRequest` and `PaginationResponse` respectively.
```

Reply:
@ipakhomov should we change this text in README too?